platform/build-faiss/add_dev.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+sudo apt-get update && sudo apt-get dist-upgrade -y
+
+# mount nvme disk on /models
+sudo mkdir /models
+sudo mkfs.xfs /dev/nvme0n1
+echo '/dev/nvme0n1  /models xfs defaults  0  2' | sudo tee -a /etc/fstab
+sudo mount -a
+sudo chmod 777 /models
+
+# Add pointers to large data dirs into the 'ubuntu' user $HOME
+mkdir /models/cache
+mv ~/.cache ~/.cache.orig
+ln -s /models/cache ~/.cache
+mkdir /models/dev
+ln -s /models/dev
+mkdir /models/local
+ln -s /models/local ~/.local
```

Comment:
The goal of this script is a little mysterious. It appears like it is meant to create local storage for development? Is that `faiss` specific? If not, could you pull this file out, and put it in a different PR? Also, a comment at the top of the file describing the purpose would help.